tasks/setup_requirements.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: Gather facts
+  setup: !!map {}
```

Comment:
Yes, the `setup` is ansible module, and it can take some parameters, but I don't want to pass any parameter, so I am giving empty dictionary instead.

Because if I leave it as
```
- name: Gather facts
  setup
```
it will result as
```
[
  {
    "name": "Gather facts".
    "setup" : None,
  }
]
```
But I want this:
```
[
  {
    "name": "Gather facts".
    "setup" : {},
  }
]
```

I don't know whether ansible can consume both versions, I just wanted to be explicit.